### PR TITLE
Some modifications in code.

### PR DESCRIPTION
**Pull request by butnaru1**

### CODE REVIEW

File: VictoriaBankFinalBeta.rb [hunk 1]
```diff
@@ -43,35 +44,35 @@
...
+    div.search('div.month-delimiter').each do |month_div|
       month_year = month_div.text.strip
-      div.search('div[@class="day-operations"]').each do |span|
-        span.search('div[@class="day-header"]').each do |day|
+      div.search('div.day-operations').each do |span|
+        span.search('div.day-header').each do |day|
           day_ = day.text.strip.gsub(/[^\d]/, '')
         end
-        span.search('ul[@class="operations-list"]/li').each do |list|
+        span.search('ul.operations-list/li').each do |list|
```
Comment (Stepandanci) [inline VictoriaBankFinalBeta.rb]: You use CSS selectors here. Try this structure.
```ruby
span.css('ul.operations-list li').each do |list|

File: VictoriaBankFinalBeta.rb [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+sleep 2
 
 # Fetch and parse HTML document
 doc = Nokogiri::HTML(browser.html)
 # Collect information about accounts
 accounts_arr = []
+account_struct = Struct.new(:name, :balance, :currency, :nature)
 nature, name, currency, nature = ''
-doc.css('//div[@class="contracts"]/div/div').each do |div|
+doc.css('div.contracts/div/div').each do |div|
```
Comment (Stepandanci) [inline VictoriaBankFinalBeta.rb]: remove `/div/div` from selector

File: VictoriaBankFinalBeta.rb [hunk 3]
```diff
@@ -34,7 +35,7 @@
       third_div.search('span').each do |span|
         if span.attr('class') == 'amount'
           balance = span.text.strip
```
Comment (Stepandanci) [inline VictoriaBankFinalBeta.rb]: `balance` should have **Float** type

```ruby
balance = span.text.strip.to_f
```

File: VictoriaBankFinalBeta.rb [hunk 4]
```diff
@@ -82,21 +83,21 @@
             money = cell.text.strip
             amount = money.to_s.scan(/\+?\d+\.\d+/)
```
Comment (Stepandanci) [inline VictoriaBankFinalBeta.rb]: Amount should has **Float** type

`money` variable is already string no need to call `.to_s` method

File: VictoriaBankFinalBeta.rb [hunk 5]
```diff
@@ -43,35 +44,35 @@
...
-        span.search('div[@class="day-header"]').each do |day|
+      div.search('div.day-operations').each do |span|
+        span.search('div.day-header').each do |day|
           day_ = day.text.strip.gsub(/[^\d]/, '')
         end
-        span.search('ul[@class="operations-list"]/li').each do |list|
+        span.search('ul.operations-list/li').each do |list|
           list.search('span').each do |cell|
             time = cell.text.strip if cell.attr('class') == 'history-item-time'
             date = "#{day_} " + "#{month_year}" + "#{time}"
```
Comment (Stepandanci) [inline VictoriaBankFinalBeta.rb]: Need to convert `date` value to `json` format and without time.
`2019-10-02`

Comment (butnaru1) [reply VictoriaBankFinalBeta.rb]: I changed it. Something else?

Comment (Stepandanci) [reply VictoriaBankFinalBeta.rb]: Write tests for your code. 
I opened an issue.
It is the last task. 
Good luck.
https://github.com/butnaru1/TestTask/issues/6

File: VictoriaBank_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+require 'rspec'
+require_relative 'VictoriaBank'
+
+describe 'VictoriaBank' do
+  it 'parse_accounts return value an array' do
+    victoriabank = VictoriaBank.new
+    expect(victoriabank.parse_accounts("#{File.open('Cards and accounts.html').read}")).to be_an_instance_of(Array)
```
Comment (Stepandanci) [inline VictoriaBank_spec.rb]: You should show an example of account data (name, balance, currency, nature).

File: VictoriaBank_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  it 'parse_transactions return value an array' do
+    victoriabank = VictoriaBank.new
+    account = 'MD22VI225931303201646102'
```
Comment (Stepandanci) [inline VictoriaBank_spec.rb:25]: You should show an example of transaction data (date, description, amount, currency, account_name).

**Comment by Stepandanci** [comment]
Remove useless files
- VictoriaBankFinalBeta.rb
Operations history_files like `.png` `.css` `.js`

Leave only
- VictoriaBank.rb
- VictoriaBank_spec.rb
- Transaction.rb 
- Account.rb 
- Cards and accounts.html
- Operations history.html

**Comment by butnaru1** [comment]
If I delete Operations history_files automat is deleted and html file

**Comment by Stepandanci** [comment]
1. press F12 
2. find element by id `contracts-cards-accounts-page` 
3. right click
4. copy outerHTML
5. save in `accounts.html`

accounts_html

**Comment by Stepandanci** [comment]
Do the same for `transactions`.
copy outerHTML off element `div.operations` on transactions page and save in `transactions.html`

It will be easy to read your pull request without 16 files for other developers.

**Comment by butnaru1** [comment]
Ok! Thank You!

**Pull request merged**